Makefile

```diff
@@ -87,25 +87,50 @@ ALL_FILES_WIN=\
     batavia\core\Frame.js \
     batavia\core\Generator.js \
     batavia\core\PYCFile.js \
-    batavia\VirtualMachine.js \
+    batavia\VirtualMachine.js
 
-.PHONY: all clean
+EXTRA_FILES=\
+		batavia/modules/stdlib/colorsys.js \
+		batavia/modules/stdlib/copyreg.js \
+		batavia/modules/stdlib/this.js
 
-all: batavia.js batavia.min.js
+EXTRA_FILES_WIN=\
+		batavia\modules\stdlib\colorsys.js \
+		batavia\modules\stdlib\copyreg.js \
+		batavia\modules\stdlib\this.js
+
+
+all: batavia-all.js batavia-all.min.js batavia.js batavia.min.js
+
+.PHONY: all clean stdlib
+
+stdlib:
+	python compile_stdlib.py
```

Comment:
The Python versioning is an important point. My original intention was to release major.minor that mirrored Python versions, with a .release that was specific to batavia, and always tracked the most recent (or as recent as possible) CPython point release. So, the first release of Batavia would be 3.4.1, tracking CPython 3.4.5.

Once we reach a point of partial completeness, we might need to start actively working on the individual branches. This will be especially important for the 3.6 branch, because they've completely changed the bytecode format.